config/i18n.json

```diff
@@ -652,7 +652,9 @@
     "ClassTypeMissing": "You do not have a {{className}} to create a loadout for",
     "ClearSection": "Remove all",
     "ClearLoadoutParameters": "Remove Loadout Optimizer settings",
-    "ClearSpace": "Move other items away",
+    "ClearSpace": "Move others away",
+    "ClearSpaceWeapons": "Move other weapons away",
+    "ClearSpaceArmor": "Move other armor away",
```

Comment:
These are from the D1 loadout editor, which I didn't want to spend a lot of time on